src/utility/HCI.cpp

```diff
@@ -112,6 +112,15 @@ void HCIClass::poll(unsigned long timeout)
 
   while (HCITransport.available()) {
     byte b = HCITransport.read();
+	
+	if (sizeof(_recvBuffer) <= _recvIndex)
```

Comment:
```suggestion
	if (_recvIndex >= sizeof(_recvBuffer))
```
Since the size of the _recvBuffer is constant this is much easier to read ...